apps/pagetour-sdk/src/managetours/all-tours-modal.html

```diff
@@ -8,6 +8,7 @@
   <div class="tour-dialog">
     <div class="tour-modal">
       <form id="tour-form" method="" onsubmit="return false;">
+        <span id="spanToolTipUrlCopyMsg" style="display:none;position:absolute;background-color:black;color: #fff;z-index:1;padding: 5px;font: size 13px !important;">Copied to clipboard!</span>
```

Comment:
can we move from inline css to class

Reply:
Thanks Pratik. I have moved the inline css to the existing file pagetour-styles.scss